salt/modules/yumpkg.py

```diff
@@ -122,12 +124,12 @@ def _get_hold(line, pattern=__HOLD_PATTERN, full=True):
     dnf ==> vim-enhanced-2:7.4.827-1.fc22.*
     """
     if full:
-        if _yum() == "dnf":
+        if _yum() == "dnf" or _yum() == "dnf5":
```

Comment:
```suggestion
        if _yum() in ("dnf", "dnf5"):
```
Single call to `_yum()`